DAQ_FW/lib/Log/Log.cpp

```diff
@@ -1,5 +1,11 @@
 #include "Log.h"
```

Comment:
To be consistent, it's much better to have the file names be the same as the namespace/class names. Let's rename the Log directory, cpp, and h files to `Logger`